selfdrive/common/visionbuf_ion.c

```diff
@@ -137,6 +138,9 @@ void visionbuf_sync(const VisionBuf* buf, int dir) {
 }
 
 void visionbuf_free(const VisionBuf* buf) {
+  clReleaseMemObject(buf->buf_cl);
+  munmap(buf->addr, buf->len + PADDING_CL);
```

Comment:
What do you think of saving the actual length in the buf struct as well? If we ever change the padding computation this will cause bugs.